index.html

```diff
@@ -36,6 +36,10 @@
             <a class="dropdown-item" href="html/projects.html#Nikhil">Nikhil</a>
           </div>
         </li>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="../html/videos.html">Videos</a>
```

Comment:
make sure here its not ../html/videos.html but just html/videos.html check comment above for reasoning.